### DISCUSSION TITLE
Change the home page of the Grafana icon?

### DISCUSSION THREAD

**Discussion by ScottAndrewsNorthWales** [grafana/grafana#60858] [General]
Hi everyone,

we need the public to use our Grafana environment and we have auto-login (using Organisations) enabled.  It works really well.  The problem we have is that if someone clicks the Grafana icon on the top left it goes to https://datagraff.secure.conwy.gov.uk/login but we actually want it to go https://datagraff.secure.conwy.gov.uk.
![2022-12-30 16_54_43-Window](https://user-images.githubusercontent.com/112002362/210094907-3d9542f4-9c76-4d4f-b652-ab80e9acbc04.png)

Is there a way to customise this?  I have looked on here https://grafana.com/docs/grafana/latest/setup-grafana/configure-grafana/enterprise-configuration/ and I can see information about changing the logo and changing the home page once people have logged in.  

Does anyone have any insights?

Thanks as ever for reading and your time.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
